.travis.yml

```diff
@@ -1,40 +1,30 @@
 language: python
 
-python:
-    - "2.7"
-    - "3.4"
+sudo: false
 
 env:
-    - "PYWR_SOLVER=glpk"
-    - "PYWR_SOLVER=lpsolve"
+    matrix:
+        - CONDA_NPY=111  CONDA_PY=27
+        - CONDA_NPY=111  CONDA_PY=34
```

Comment:
I don't see why not. I don't see any benefit in testing other numpy versions though.